examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js

```diff
@@ -107,12 +107,17 @@ export default class MeshLayer extends SimpleMeshLayer {
 
     const shaders = this.getShaders();
 
+    const customDefines = {};
+    if (mesh.attributes.uvRegions) {
```

Comment:
I am not super fond of using defines. I think the overhead of always using a region is small, we could just use a default constant region attribute instead.